data/tutorial/03_getting-started.md

```diff
@@ -73,13 +73,13 @@ module Mem_Store =
         (Mem_schema)
 ```
 
```

Comment:
It's not clear where to apply this example and others formatted like it. It doesn't seem like it's to be tested in the CLI, so I suspect it's to amend a file....but which file?

Same question for others throughout.